ghost/i18n/locales/hi/portal.json

```diff
@@ -0,0 +1,60 @@
+{
+    "{{discount}}% discount": "{{chhoot}}% chhoot",
+    "{{trialDays}} days free": "{{trialDays}} din muft",
+    "A login link has been sent to your inbox. If it doesn't arrive in 3 minutes, be sure to check your spam folder.": "aapake inaboks mein ek login link bhej diya gaya hai. yadi yah 3 minat mein nahin aata hai, to apane spaim pholdar kee jaanch avashy karen.",
+    "Account": "khaata",
+    "Account settings": "akaunt seting",
+    "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "shulk pareekshan samaapt hone ke baad, aapake dvaara chune gae star ke lie aapase niyamit mooly liya jaega. aap isase pahale hamesha radd kar sakate hain.",
+    "Already a member?": "kya pahale se hee sadasy hain?",
+    "Back": "peechhe",
+    "Back to Log in": "login par vaapas jaen",
+    "Cancel subscription": "sadasyata radd",
+    "Cancellation reason": "radd karane ka kaaran",
+    "Choose a different plan": "koee bhinn yojana chunen",
+    "Choose your newsletters": "apane nyoozaletars chunen",
+    "Close": "band keejiye",
+    "Comments": "tippaniyaan",
+    "Confirm": "pushti karana",
+    "Continue": "jaaree rakhana",
+    "Delete account": "khaata hata do",
+    "Don't have an account?": "khaata nahin hai?",
+    "Email": "eemel",
+    "Email preference updated.": "eemel vareeyata apadet kee gaee.",
+    "Email preferences": "eemel praathamikataen",
+    "Emails": "eemel praathamikataen",
+    "Emails disabled": "eemel aksham",
+    "Get help": "madad len",
+    "Get notified when someone replies to your comment": "jab koee aapakee tippanee ka javaab de to soochana praapt karen",
+    "Give feedback on this post": "is post par pratikriya den",
+    "Less like this": "is tarah kam",
+    "Manage": "prabandhit karana",
```

Comment:
Will have to check the context in which these words are used. But looks ok for now